activesupport/lib/active_support/concern.rb

```diff
@@ -137,6 +141,22 @@ def included(base = nil, &block)
       end
     end
 
+    # Define class methods from given block.
+    # You can define multiple +class_methods+ blocks.
+    #   module Example
+    #     extend ActiveSupport::Concern
+    #     class_methods do
+    #       def foo; print 'foo'; end
+    #     end
+    #
+    #     class_methods do
+    #       def bar; foo; print 'bar'; end
+    #     end
+    #   end
+    #
+    #   Buzz = Class.new.include(Example)
+    #   Buzz.bar
```

Comment:
Just enough one line.
```suggestion
    #   Buzz.bar # => "foobar"
```